src/lowlevel.h

```diff
@@ -0,0 +1,55 @@
+// Copyright (c) 2021 The Dogecoin Core developers
+// Distributed under the MIT software license, see the accompanying
+// file COPYING or http://www.opensource.org/licenses/mit-license.php.
+
+//Here be dragons, wizards, witches and master of the dark arts. 
+//Careful thou who wonders 'round these parts without magic spells,
+//mana potions and doggy treats...especially magical doggy treats.
+
+
+//Only define this header on x86
+#if ( defined(i386)     || defined(__i386)  || defined(__i386__) || \
+      defined( __386)   || defined(_X86_)   || defined( _M_I86)  || \
+      defined(__i386__) || defined(__X86__) || defined(__x86_64) ) 
+
+    #ifndef LOWLEVEL_H
+        #define LOWLEVEL_H
```

Comment:
This header guard currently also serves as a functionality flag in `random.cpp`, recommend to define a separate functionality flag, eg `#define USE_RDSEED`, right underneath this, as that would keep flags serving a single purpose.

Reply:
Good suggestion. I will do that.

Reply:
I've reopened this comment. The new code can still be optimized but we can do that separately.